kubernetes/istio-rbac/istio-rbac.yaml

```diff
@@ -0,0 +1,21 @@
+# Permissions and roles for istio
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: istio-manager-service-account
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: istio-manager-admin-role-binding
+subjects:
+- kind: ServiceAccount
+  name: istio-manager-service-account
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Agreed, I'm testing with admin for the namespace only - and using separate accounts for 
proxy and manager. 

Ideally ingress will only have read access, plus restricted access to secrets, and manager
may need read/write access - I don't know if it can be narrowed to specific resources.

I think most important is restricting ingress - since it's exposed to outside access, and 
second priority to make sure the sensitive ingress TLS certs are only visible to the ingress
server (since access to them allows intercepting all ingress traffic)